### ISSUE TITLE
Error when casting Float to Double and vice versa

### ISSUE THREAD

**Issue by WouterSpekkink** [gephi/gephi-plugins#119]
I noticed that it is no longer possible to cast a Float to a Double or vice versa. For example, if I import coordinates as floats in Gephi, and then try to lay them out with the Geo-Layout plugin, I get the error report as shown in the attachment.

I believe this was not an issue in the previous version of Gephi.

![errorreport](https://cloud.githubusercontent.com/assets/6826078/15643985/89241624-2650-11e6-8a49-ec270c49094a.png)

**Comment by eduramiba** [comment]
Thanks for the report, I think it used to store primitives instead of wrappers. Now the good way is to check that the column is a type of Number and do something like ```((Number) object).doubleValue();```

**Comment by WouterSpekkink** [comment]
So is this something we should deal with on the plugin side? I have the same problem with the event graph layout, for which a pull request is running. If there is something I should change there, I could probably do another commit this evening.

**Comment by eduramiba** [comment]
Sure, ideally you should check that the plugin works with any number type column. Something that might also work is casting to primitive ```double``` instead of wrapper.

**Comment by WouterSpekkink** [comment]
Okay, I think I fixed it for the event-graph-layout, following your suggestion. I just tested it by running Gephi with the plugin, and this time importing something as a float didn't cause an error. I'm not sure if my solution is very pretty, but it would work for the Geo-Layout as well. I basically first retrieve the number from the user-supplied column as a Number variable. In my case the variable is a double called 'ord', and it is retrieved from a column called 'order.' I then use the following code (n refers to a node, as I am using a for-loop to iterate through all nodes in the graph):

`ord = ((Number) n.getAttribute(order)).doubleValue();`

Works very well. I didn't know the trick with enclosing the entire expression in parentheses yet.
